app/models/mixins/per_ems_worker_mixin.rb

```diff
@@ -97,7 +97,7 @@ def queue_name_for_ems(ems)
       return "generic" if ems.kind_of?(Host) && ems.acts_as_ems?
```

Comment:
hmm, do we need this conditional?   Maybe we need to add this method to host too since host.rb's acts_as_ems seems to indicate we use this for hyperv.